packages/wonder-blocks-tokens/src/build/generate-css-variables.ts

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env -S node -r @swc-node/register
+import fs from "fs";
+import path from "path";
+import ancesdir from "ancesdir";
+import {generateTokens} from "../internal/generate-tokens";
+
+const THEMES_DIR = "../theme/color";
+
+/**
+ * Process all themes in the theme directory.
+ *
+ * This step will read all the theme files in the theme directory and generate
+ * the CSS variables for each theme.
+ */
+function processThemeCollection() {
+    return fs.readdirSync(path.resolve(__dirname, THEMES_DIR)).map((file) => {
+        // Remove the file extension
+        const filename = file.split(".")[0];
+        // eslint-disable-next-line @typescript-eslint/no-require-imports
+        const {default: themeObject} = require(`../theme/color/${filename}`);
```

Comment:
Can you use `THEMES_DIR` here to make it easy to update?

```js
const {default: themeObject} = require(`${THEMES_DIR}/${filename}`);
```

Reply:
Good catch! I'll replace it :)